chef/cookbooks/neutron/templates/default/neutron.conf.erb

```diff
@@ -216,11 +216,7 @@ dhcp_agents_per_network = <%= @network_nodes_count %>
 # worker thread in the current process.  Greater than 0 launches that number of
 # child processes as workers.  The parent process manages them.
 # api_workers = 0
-# This is workaround for lp#1354226
-# https://bugs.launchpad.net/neutron/+bug/1354226
-<% unless @core_plugin == "vmware" -%>
-api_workers = <%= [node["cpu"]["total"], 2, 4].sort[1] %>
-<% end -%>
+api_workers = 0
```

Comment:
non zero api workers is not experimental, but its helpful in removing the memory overhead..